src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -152,127 +226,126 @@ These variables can be set in your local environment shell before running and th
 |`CHE_PORT` |The port the Che server will run on and expose in its container for your clients to connect to.
 |===
 
-Che depends upon Docker images. We use Docker images to:
+Che depends on Docker images. Docker images are used to:
+
+* Provide cross-platform utilities within the CLI. For example, to perform a `curl` operation, you use a small Docker image to perform this function. This is done as a precautionary measure because many operating systems (like Windows) do not have curl installed.
```

Comment:
A number of Linux distributions also do not have `curl` by default. I think "(like Windows)" is unnecessary.

Also, the last mention of `curl` is unstyled here. :)